packages/components/src/components/post-header/post-header.tsx

```diff
@@ -333,11 +333,6 @@ export class PostHeader {
         style={{ '--post-header-navigation-current-inset': `${mobileMenuScrollTop}px` }}
       >
         <div class="mobile-menu" ref={el => (this.mobileMenu = el)}>
-          <div class="navigation-target-group">
```

Comment:
Moving around markup here is a breaking change. Is there another way to make this work?